nengo_loihi/splitter.py

```diff
@@ -263,13 +264,11 @@ def split_host_to_chip(networks, conn):
     # scale the input spikes based on the radius of the
     # target ensemble
     seed = networks.original.seed if conn.seed is None else conn.seed
-    transform = nengo.dists.get_samples(
-        conn.transform,
-        n=conn.size_out,
-        d=conn.size_mid,
-        rng=np.random.RandomState(seed=seed))
+    weights = conn.transform.sample(rng=np.random.RandomState(seed=seed))
     if isinstance(conn.post_obj, nengo.Ensemble):
-        transform = transform / conn.post_obj.radius
+        weights = weights / conn.post_obj.radius
+    transform = copy.copy(conn.transform)
+    type(transform).init.data[transform] = weights
```

Comment:
That looks weird. If there's no way to do that nicer, maybe we're missing some sort of functionality in `FrozenObject` or `Transform`.

Reply:
It might be nice to have a `copy` function on `FrozenObject` that let's you make a copy, but pass any constructor arguments to change those values. Could be useful for users if they've got some object like a neuron type, and want to play around with some specific parameter while keeping everything else constant. I'll make a Nengo issue, and we can leave this as-is for now.

Reply:
nengo/nengo#1509